src/styles/components/_commander-option.scss

```diff
@@ -1,12 +1,24 @@
 .c-commander-option {
     border: 0;
     padding: 0;
-    background-color: $color-black;
-    color: $color-white;
+    background-color: rgb(51 38 1 / 17%);
+    color: $color-black;
+    width: 250px;
+    padding: 16px;
+    border-radius: 12px;
+    width: 250px;
+    text-align: left;
+
+    img {
```

Comment:
I'd like to avoid nesting whenever possible, and I'd also like to avoid using tag selectors.  I've run into some Specificity Wars problems when that starts to get out of hand.  Would you be able to make a classname called `c-commander-option__image` so that it fits the [BEM standard](http://getbem.com/)?